### ISSUE TITLE
Is it possible to toggle the menu on certain pages?

### ISSUE THREAD

**Issue by Phorce** [modularcode/modular-admin-html#53]
I have a question, more than an issue. 

Is it possible to toggle the menu and show the burger icon rather than the full menu on certain pages? Similar to what happens when viewing the page on an iPad?

**Comment by modularcoder** [comment]
@Phorce sure, but It's not implemented out of the box.

I suggest adding an additional class ```sidebar-closed``` to .app element, which will adjust all the styles as per requirement. And the toggle button will just toggle that class

**Comment by jesephan** [comment]
@modularcoder adding `sidebar-closed` class seems not working. Hope you guys can also add feature where we can toggle shrink the sidebar on button click like most of dashboard where only icons are showing :)

**Comment by modularcoder** [comment]
@jesephan yep, it's not implemented so far :) For now It needs to be done manually.

I'm working on V2 version which should support sidebar toggle / collapse functionality.

https://github.com/modularcode/modular-admin-html/tree/v2-dev
https://modularcode.io/modular-admin-html-demo/

**Comment by jesephan** [comment]
@modularcoder That's nice I will surely switch to it when its ready lol by the way do have estimated release date for the v2?

**Comment by modularcoder** [comment]
@jesephan not so far. And there's quite lot of work to do yet :/

**Comment by elyeszitari** [comment]
Hi @modularcoder in V2 version, sidebar can be swuished to show icons menu only?

**Comment by iwalucas** [comment]
Has someone implemented a workaround for this?
